dashboard/app/app_test.go

```diff
@@ -317,6 +317,51 @@ var testConfig = &GlobalConfig{
 				},
 			},
 		},
+		"fs-bugs-reporting": {
+			AccessLevel: AccessPublic,
+			Key:         "fspublickeypublickeypublickey",
+			Clients: map[string]string{
+				clientPublicFs: keyPublicFs,
+			},
+			Repos: []KernelRepo{
+				{
+					URL:    "git://syzkaller.org/fs-bugs.git",
+					Branch: "fs-bugs",
+					Alias:  "fs-bugs",
+				},
+			},
+			Reporting: []Reporting{
+				{
+					Name:       "wait-repro",
+					DailyLimit: 1000,
```

Comment:
This step makes no external interactions, so the daily limit can likely be enormous (maintainers, please suggest an appropriate "large" value?)

Reply:
That is the configuration for Go tests. Once the PR is merged, I'll make the corresponding changes to our production config, where we do have proper limits and more filters.